src/Sulu/Bundle/AdminBundle/Resources/js/components/Backdrop/Backdrop.js

```diff
@@ -4,14 +4,14 @@ import Portal from 'react-portal';
 import React from 'react';
 import backdropStyles from './backdrop.scss';
 
-export default class Backdrop extends React.PureComponent {
-    props: {
-        isOpen: boolean,
-        /** When set to false the backdrop renders transparent. */
-        isVisible: boolean,
-        onClick?: () => void,
-    };
+type Props = {
```

Comment:
@chirimoya @mmsbrggr @3rfan We have to decide how to write the props resp. type in general now. I see the following options here:

1. We inline this objection definition in the generic like `React.PureComponent<{isOpen: boolean}>`. I think that is not desirable, since it gets hard to read if there are a few more props.
2. We transfer this type to the `types.js` file. That would mean that it is not so easy to just find the corresponding props to the component.
3. We get rid of the `types.js` file at all and move all types to the "correct" component.
4. We keep the `types.js`file and make an exception for the prop types.